knack_load_fees.py

```diff
@@ -91,41 +91,24 @@ def map_row(row):
     return new_row
 
 
-def is_dupe_row_error(res):
-    """Checks a Knack API `400` response to see if it's caused by a known issue of attempting
-    to insert duplicate records. There duplicate rows in our AMANDA data due to errors
-    in SQL query. See: https://github.com/cityofaustin/atd-data-tech/issues/10447.
-
-    The response JSON we're handling looks like this:
-    {
-        'errors':
-        [
-            {
-                'field': 'field_285',
-                'type': 'unique',
-                'message': 'Fee Number must be unique. "13646573" is already being used.'
-            }
-        ]
-    }
-
-    Args:
-        res (requests.models.Response): a requests Response object from a Knack API call
```

Comment:
this should no longer be necessary thanks to our improved query and new combined ID column